Sources/GEOSwift/GEOS/GEOSHelpers.swift

```diff
@@ -44,6 +61,10 @@ func makeCoordinateSequence(with context: GEOSContext, points: [Point]) throws -
                 GEOSCoordSeq_destroy_r(context.handle, sequence)
                 throw GEOSError.libraryError(errorMessages: context.errors)
         }
+        if let z = point.z, GEOSCoordSeq_setZ_r(context.handle, sequence, UInt32(i), z) == 0 {
+            GEOSCoordSeq_destroy_r(context.handle, sequence)
+            throw GEOSError.libraryError(errorMessages: context.errors)
+        }
```

Comment:
If dimensionality determined above is 3 but some other point in the input has dimensionality 2, should we set some default value for Z?